### PR TITLE
feat: add PROPS_TEMPLATE, raw, highlight

### PR DESCRIPTION
**Pull request by daybrush**

### CODE REVIEW

File: src/code/utils.ts [hunk 1]
```diff
@@ -74,12 +83,16 @@ export function convertFunction(func: string, type: CODE_TYPE, comment: string)
...
+            return includeComment(removeBracket(toReactArrow(func)), "method", comment);
+        case CODE_TYPE.SVELTE_ARROW:
+            return includeComment(removeBracket(toCustomEvent(toSvelteArrow(func))), "method", comment);
         default:
             return func;
     }
 }
 export function previewFunction(func: string) {
-    return (type: CODE_TYPE, comment = "") => {
+    return (type: CODE_TYPE, comment = "", ) => {
```
Comment (mixed) [inline src/code/utils.ts:95]: Could you explain why add `,` ?

File: src/index.tsx [hunk 2]
```diff
@@ -21,85 +22,21 @@ export const preview = (parameter: any) => {
...
-                value = p[name];
-            }
-
-            if (typeof value === "object") {
-                value = JSON.stringify(value);
-            }
-            return "" + prev + next  + value;
-        }, "");
-        return dirty;
+        return knobs[${JSON.stringify(name)}];
```
Comment (mixed) [inline src/index.tsx:29]: I think that `JSON.stringify` is unnecessary. `name` is string.

File: README.md [hunk 3]
```diff
@@ -109,6 +109,114 @@ const inst = new Instance({
 ```
 
 
+
+#### Highlight
```
Comment (WoodNeck) [inline README.md:113]: How about adding an HTML highlight tutorial here? (with )
I think it's good to show that we also support that.

File: src/index.tsx (hunk 2 above)
Comment (daybrush) [reply src/index.tsx:29]: @mixed 

The reason is to put quotes.

**Pull request merged**